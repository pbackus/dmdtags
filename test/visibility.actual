!_TAG_FILE_SORTED	1	/0=unsorted, 1=sorted, 2=foldcase/
visibility	test/visibility.d	1;"	M
x	test/visibility.d	1;"	v	file:
y	test/visibility.d	2;"	v
