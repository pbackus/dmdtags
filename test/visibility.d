private { public { private int x; } }
/* public */ int y;
