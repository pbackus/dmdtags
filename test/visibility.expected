!_TAG_FILE_SORTED	1	/0=unsorted, 1=sorted, 2=foldcase/
!_TAG_FILE_ENCODING	utf-8	//
visibility	test/visibility.d	1;"	M
x	test/visibility.d	1;"	v	file:
y	test/visibility.d	2;"	v
